projects/manager/src/app/datasets/dataset-list/dataset-list.component.html

```diff
@@ -1,20 +1,33 @@
 <div class="container">
-    <div>
-        <mat-form-field appearance="fill">
-            <mat-label>Filter by name</mat-label>
-            <input matInput type="text" (input)="onSearchChange($event)" />
-        </mat-form-field>
-    </div>
-    <cdk-virtual-scroll-viewport class="bottom" [itemSize]="itemSizePx" (scrolledIndexChange)="onScrolledIndexChange($event)">
+    @if (addedDataset; as item) {
+        <button mat-stroked-button class="browseButton" (click)="backToAllDatasets()">
+            <mat-icon>chevron_left</mat-icon>Back to all datasets
```

Comment:
When I delete the dataset, I am still in this view.

Reply:
is it bad?